packages/library-sync/src/commonMain/kotlin/io/realm/mongodb/sync/NetworkTransport.kt

```diff
@@ -0,0 +1,3 @@
+package io.realm.mongodb.sync
```

Comment:
Internal? Or is this actually needed by the user for setup?

Reply:
I believe it should be internal, yes.

Reply:
Now that I think of it, I don't know if it can be internal since it will be the C-API that will be calling the `sendRequest` function. I guess we will have to try whether it can be done 🤷